console.js

```diff
@@ -22,11 +22,11 @@ const convertObjectFunctionsToAsync = serviceMethods => {
   return asyncServiceMethods;
 };
 
-orm.init().then(() => {
+Promise.resolve().then(() => {
```

Comment:
There is no need of this, anyways i'll update this script after this pr is merged